### PR TITLE
Documentation on active learning

### PR DESCRIPTION
**Pull request by J535D165**
See #183 

PR by @Sasafrass

### CODE REVIEW

**Comment by J535D165** [comment]
Looks promising. Do we have some image(s) we can add?

**Comment by Sasafrass** [comment]
I refer to "Figure 1" on the github homepage. I'd say we can reuse that image for the active learning section as well

File: docs/source/activelearning.rst [hunk 1]
```diff
@@ -2,3 +2,27 @@ Active Learning
 ~~~~~~~~~~~~~~~
 
 Introduction of Active Learning on (academic) literature.
+
+Active learning denotes the scenario in which a reviewer actively provides classifications for references which are presented by the model [3,4]. Subsequently, the model retrains based on the new input. When it has retrained, it decides on a new reference to be annotated and it shows this to the reviewer. In this way, the annotated dataset starts out small and iteratively grows in size [4]. In the case of Automated Systematic Review, this process is necessary to provide some initial classifications to the model, because the reviewer starts with a dataset without annotations.
```
Comment (bagheria) [inline docs/source/activelearning.rst]: I think "the model" should be "the learning model" or "the machine learning model", or "the classifier". Am I miss something here? I don't have previous paragraphs.

Comment (Sasafrass) [reply docs/source/activelearning.rst]: Agreed! Maybe call it "machine learning model" once and then refer to it as the model? Or keep referring to it as the "machine learning model"? Thanks for the feedback!

Comment (GerbrichFerdinands) [inline docs/source/activelearning.rst]: I think that the first sentences could be a bit more clear. If I may, here is a suggestion: 

> Active learning denotes the scenario in which the reviewer is labeling references that are presented by a machine learning model. The machine learning model learns from the reviewers labeling decisions and uses this knowledge in selecting the reference that will be presented to the reviewer next.

Comment (GerbrichFerdinands) [reply docs/source/activelearning.rst]: I'd say calling it 'machine learning model' the first time and 'model' later on is fine!

Comment (Sasafrass) [reply docs/source/activelearning.rst]: So replace the sentences "active learning denotes... to the reviewer" with your suggestion? Sounds like a good improvement to me!

**Comment by J535D165** [comment]
Thanks for your contribution @Sasafrass!

**Pull request merged**